### ISSUE TITLE
Two solutions for python on osx

### ISSUE THREAD

**Issue by pelson** [conda-forge/python-feedstock#1]
```
Warning: 2 possible package resolutions (only showing differing packages):
  - python-3.5.1-0.tar.bz2, sqlite-3.9.2-0.tar.bz2
  - python-3.5.0-1.tar.bz2, sqlite-3.11.1-0.tar.bz2
```

We will want to move our pinning forwards to resolve this (there is no *major* urgency on this though)

**Comment by jakirkham** [comment]
I think @ocefpaf has some problems with that. In particular, something is off with `libspatialite`.

**Comment by ocefpaf** [comment]
Indeed. `libspatialite` has two test failures with `sqlite-3.11.1`. Maybe that is nothing major and we can skip those tests, but I am not an expert. With that said, if `sqlite-3.11.1` is important, let's pin to that and I can removed `libspatialite` on `gdal` until we figure this out.

**Comment by jjhelmus** [comment]
I think this has been solved when the Python 3.5 pinnings were updated.  Can this be closed?
